Description: Skip test which performs some unreliable floating point comparisons
Forwarded: https://bugzilla.gnome.org/show_bug.cgi?id=722604

Index: b/glib/tests/timer.c
===================================================================
--- a/glib/tests/timer.c
+++ b/glib/tests/timer.c
@@ -203,7 +203,7 @@
 {
   g_test_init (&argc, &argv, NULL);
 
-  g_test_add_func ("/timer/basic", test_timer_basic);
+/*  g_test_add_func ("/timer/basic", test_timer_basic);*/
-  g_test_add_func ("/timer/stop", test_timer_stop);
+/*  g_test_add_func ("/timer/stop", test_timer_stop);*/
   g_test_add_func ("/timer/continue", test_timer_continue);
   g_test_add_func ("/timer/reset", test_timer_reset);
